Src/PerformanceCollector/Filtering/AccumulatedValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Collections.Concurrent;
+
+    /// <summary>
+    /// Accumulator for operationalized metrics.
+    /// </summary>
+    internal class AccumulatedValue
+    {
+        public readonly ConcurrentStack<double> Value = new ConcurrentStack<double>();
```

Comment:
Class is calles `AccumulatedValue`. Rename to `AccumulatedValues` at least. Will aggregation differ depending on the order? Why `Stack`?